js/interface.js

```diff
@@ -751,9 +751,18 @@ function insertItem(item, isFolder) {
 }
 
 function search(type, term) {
-  var query = {
-    name: term
-  };
+  var query = {};
+  // If the user put anything except a numbers _.toNumber function will return a NaN result
```

Comment:
empty line before comments

Reply:
@squallstar 
added a line before the comment and slightly improved code.